docs/usage.rst

```diff
@@ -2,7 +2,7 @@ Usage
 -----
 
 `AsynchronousLogstashHandler` is a custom logging handler which
-sends Logstash messages using UDP and TCP. For example:
+sends Logstash messages using UDP and TCP. (`SynchronousLogstashHandler` is a synchronous alternative) For example:
```

Comment:
:+1:  for updating docs!
Could you break the long lines so it would better align with the rest of the docs?

Reply:
Done.